code/modules/research/research.dm

```diff
@@ -306,6 +307,10 @@ research holder datum.
 
 	if(current_level >= level)

```

Comment:
This is the SAME thing your trying to put in but it's already there, did you even test to see if you could actually dupe tech disks because I couldn't on the build as of today.